db_scripts/db_restore

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+if [ -z $1 ]
+	then
+		echo "No SQL script file"
```

Comment:
Prefer outputting some useful usage info in this case, eg. `echo "USAGE: $0 SQL_SCRIPT"`